include/bluetooth/bluetooth.h

```diff
@@ -664,18 +670,55 @@ struct bt_le_scan_param {
 
 	/** Scan window (N * 0.625 ms) */
 	u16_t window;
+
+	/** Scan timeout (N * 10 ms)
+	 *  Application will be notified by the scan timeout callback.
+	 *  Set zero to disable timeout.
+	 */
+	u16_t timeout;
+
+	/** Scan interval coded (long range) (N * 0.625 MS)
+	 *  Set zero to use same as 1M scanning
+	 */
+	u16_t interval_coded;
+
+	/** Scan window coded (long range) (N * 0.625 MS)
+	 *  Set zero to use same as 1M scanning
+	 */
+	u16_t window_coded;
 };
 
 /** LE advertisement packet information */
 struct bt_le_adv_info {
-	/** Advertiser LE address and type */
+	/** Advertiser LE address and type. */
```

Comment:
Document that this will be `BT_ADDR_LE_ANY` if the report comes from an anonymous advertiser